js/Humberg.js

```diff
@@ -0,0 +1,210 @@
+// Data for "Speakers" Section
+
+const meskelsData = [
+  {
+    img: '../Resource/Images/posters/benaw.jpg',
+    name: 'Habtamu Alemayehu',
+    Postion: 'Software Developer - 2023',
+    'meskel sinopsis':
+        '"The meskel celebration in Gurage may have a span of 14days depending on the area. " ',
+  },
+  {
+    img: '../Resource/Images/posters/benaw.jpg',
+    name: 'Habtamu Alemayehu',
+    Postion: 'Software Developer - 2023',
+    'meskel sinopsis':
+        '"During the slaughter ceremony the ox should fall on the right side or is flipped the other way around. " ',
+  },
+  {
+    img: '../Resource/Images/posters/benaw.jpg',
+    name: 'Habtamu Alemayehu',
+    Postion: 'Software Developer - 2023',
+    'meskel sinopsis':
+        '"This is the day that the whole of Ethiopia celebrates meskel. On this day you might see people with crosses made of ashes" ',
+  },
+  {
+    img: '../Resource/Images/posters/benaw.jpg',
+    name: 'Habtamu Alemayehu',
+    Postion: 'Software Developer - 2023',
+    'meskel sinopsis':
+        '"Meskel in Gurage is not only limited within the reigon but also the whole of Ethiopia."',
+  },
```

Comment:
- [ ] To make your application looks more professional, I think it would be better if you add different speakers and different data to each Speakers